sig-architecture/production-readiness.md

```diff
@@ -80,9 +80,28 @@ To become a reviewer:
  * Read/study previous PRR comments and production readiness responses in existing KEPs.
  * Choose some KEPs requiring PRR and perform a review. Put "shadow prod readiness review"
    in your review comments so that the assigned PRR approver knows your intent.
- * After at least one release cycle, if you have shown good judgement and quality reviews,
-   you can propose yourself as approver by submitting a PRR to add your GitHub
-   handle to the `prod-readiness-approvers` alias in [OWNERS_ALIASES].
+
+### Becoming an approver
+
+After serving as reviewer/shadow for at least one release and showing good judgement and quality reviews,
+you can propose yourself as an approver by submitting a PR to add your GitHub
+handle to the `prod-readiness-approvers` alias in [OWNERS_ALIASES].
+
+When submitting the PR, you should include references to KEPs you reviewed that demonstrated a good variety
+of different situations.
+Here is a good starting point (remember that one PR can cover multiple categories):
```

Comment:
Asking the same question I asked in the last PRR meeting - which, if any, of the below situations can be covered when you fill out the PRR section of KEP that you are driving yourself?

Reply:
I think we want this to be based on reviews, not on KEPs you have authored or are driving. @deads2k , @wojtek-t , WDYT?

Reply:
Yes - I agree.
The difference is that you're super familiar with stuff you're driving yourself and we want to see that for things that are for things that you're at least a bit less familiar with.